ci/infra/openstack/master-instance.tf

```diff
@@ -70,8 +70,8 @@ resource "openstack_compute_instance_v2" "master" {
   }
 
   security_groups = [
-    openstack_networking_secgroup_v2.common.name,
-    openstack_networking_secgroup_v2.master_nodes.name,
+    openstack_networking_secgroup_v2.common.id,
+    openstack_networking_secgroup_v2.master_nodes.id,
   ]
 
```

Comment:
Wow, with this, I realised:
1) load-balancer security group is already okay.
2) load-balancer security group is not using the common, which might or might not be okay. Assuming okay.